content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
+
+To set the current report as the default configuration when the report is accessed, check the box
+next to :guilabel:`Default filter`. If the current report should be accessible to everyone in the
+database, check the box next to :guilabel:`Share`.
+
+Finally, click :guilabel:`Save`, which saves the currently configured report, and it now appears
+beneath :guilabel:`Favorites`.
+
+Default reports
+===============
 
 Payroll
 -------
 
-Click on :menuselection:`Payroll --> Reporting --> Payroll` to display the :guilabel:`Payroll
-Analysis` report.
+Click on :menuselection:`Payroll app --> Reporting --> Payroll` to display the :guilabel:`Payroll
+Analysis` report. This report shows all the payslips generated in the last 365 days.
 
 .. image:: reporting/payroll-report.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Payroll overview report showing payroll for the last 365 days.
+
+The report can display metrics for a variety of parameters. Click the :guilabel:`Measures` box to
+view a drop-down menu with the various options to display.
 
-This shows all the payslips generated in the last 365 days. The report can display metrics for a
-variety of parameters. Click the :guilabel:`Measures` box to view a drop-down menu with the various
-options to display. The options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
+The default options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
 :guilabel:`Basic Wage for Time Off`, :guilabel:`Days of Paid Time Off`, :guilabel:`Days of
 Unforeseen Absence`, :guilabel:`Days of Unpaid Time Off`, :guilabel:`Gross Wage`, :guilabel:`Net
 Wage`, :guilabel:`Number of Days`, :guilabel:`Number of Hours`, :guilabel:`Work Days`,
 :guilabel:`Work Hours`, and :guilabel:`Count`.
 
+:guilabel:`Net Wage` is the default metric for the :guilabel:`Payroll` report.
+
 .. image:: reporting/measures.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: The various measures available to display for the Payroll Analysis report.
 
 Line chart
 ~~~~~~~~~~
 
-A line chart is the default view for the *Payroll Analysis* report. If a different view was
-selected, click on the :guilabel:`Line Chart` icon (middle icon) in the menu bar to change the view
-back to a line chart. An option to display the line chart in ascending or descending order appears
-at the end of the options.
+A line chart is the default view for the *Payroll* report. If a different view is selected, click
+the :guilabel:`Line Chart` icon in the menu bar to change the view back to a line chart.
+
+Several options are available for the line chart. Click the corresponding button to activate the
+selection, and change the way the data is presented.
+
+The various options are: :guilabel:`Stacked`, :guilabel:`Cumulative`, or in :guilabel:`Descending`
+or :guilabel:`Ascending` order. These icons appear at the end of the chart options.
```

Comment:
do you think it might be useful to expand on each of these options mean/represent?

Reply:
I like this idea- I have a few questions out with the experts (haven't heard back yet today) to see if my explanations are thorough enough.